test/cpp/util/BUILD

```diff
@@ -34,8 +34,8 @@ licenses(["notice"])  # 3-clause BSD
 cc_binary(
     name = "testso.so",
     srcs = [],
-    deps = ["//:grpc++_unsecure"],
     linkshared = 1,
+    deps = ["//:grpc++_unsecure"],
```

Comment:
This was done automatically when I saved the file on my workstation, I can undo it.